packages/database/src/api/Database.ts

```diff
@@ -37,6 +36,14 @@ import {
 
 import { Reference } from './Reference';
 
+// TODO: revert to import {FirebaseDatabase as ExpDatabase} from '@firebase/database' once modular SDK goes GA
```

Comment:
There are some unrelated changes. Can you merge or rebase with the master ?

Reply:
Rebased. These changes should be all gone now.